src/main/java/seedu/address/logic/parser/FindTransactionCommandParser.java

```diff
@@ -0,0 +1,107 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_GOOD_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+import seedu.address.logic.commands.FindTransactionCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.transaction.TransactionContainKeywordsPredicate;
+
+/**
+ * Parses input arguments and creates a new FindTransactionCommand object
+ */
+public class FindTransactionCommandParser implements Parser<FindTransactionCommand> {
+
+    public static final String BUY_KEYWORD = "buy";
+    public static final String SELL_KEYWORD = "sell";
```

Comment:
Do you think we should reuse BuyCommand.COMMAND_WORD and SellCommand.COMMAND_WORD here?
So if we ever have to change the command word we only have to do it in one place (in the respective commands)